src/workloads/query/CsiFragmentedInsertsFlat.yml

```diff
@@ -0,0 +1,183 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query-execution"
+Description: |
+  This workload compares performance of inserts into a collection with only the default _id index,
+  and in presence of the columnstore index. It uses an artificial data set with a wide overall
+  schema and narrow individual objects to model fragmented access to CSI, which clusters entries by
+  path. The data size is relatively small (1e6 documents yield ~175MB data size and ~105MB storage
+  size).
+  We would like to be able to correlate the results of this workload with the similar one that uses
+  nested data (CsiFragmentedInsertsNested.yml). Please make sure to update both when making changes.
+
+Keywords:
+- columnstore
+- wildcard
```

Comment:
Are you actually testing against wildcard here? I don't see it.

Reply:
oops, at the start we did test here against wildcard as well but since then the workload got, em, simplified :)